install/kubernetes/experimental-install.yaml

```diff
@@ -1238,7 +1238,7 @@ spec:
         k8s-app: hubble-generate-certs
     spec:
       serviceAccount: hubble-generate-certs
-      serviceAccountName: hubble-generate-certs
+      serviceAccountName: hubble-generate-certs      
```

Comment:
Not sure where it is coming from, but there seems to be some trailing whitespace introduced here.

Reply:
Fixed.